frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineDetails.tsx

```diff
@@ -40,8 +41,8 @@ const PipelineDetails: React.FC<PipelineDetailsTabProps> = ({ obj: pipeline, cus
               title={t('pipelines-plugin~Finally tasks')}
             />
             <WorkspaceDefinitionList workspaces={pipeline.spec.workspaces} />
-          </div>
-        </div>
+          </GridItem>
+        </Grid>
```

Comment:
issues on Pipeline details page
<img width="1181" alt="Screenshot 2025-05-06 at 5 17 29 PM" src="https://github.com/user-attachments/assets/4fc48fa9-e4e9-4036-98da-d5a06a3de5e8" />

Reply:
for comparison - 4.19.0-0.nightly-2025-05-05-205331
<img width="1267" alt="Screenshot 2025-05-06 at 5 19 48 PM" src="https://github.com/user-attachments/assets/f653ea06-b039-4f23-8912-ab3802f18098" />

Reply:
I am unable to reproduce:

![Uploading image.png…]()